lib/metanorma/cli.rb

```diff
@@ -71,10 +71,12 @@ def self.templates_path
       Pathname.new(Dir.home).join(".metanorma", "templates")
     end
 
-    def self.templates_path_access
-      dir = templates_path
-      dir = dir.parent while !dir.exist?
-      File.writable? dir
+    def self.writable_template_path?
+      unless templates_path && templates_path.writable?
```

Comment:
@abunashir  Are you sure that this will work for non-existing path

Reply:
Good catch!